lib/pages/chat/events/message/message.dart

```diff
@@ -856,3 +861,37 @@ class ReplyIconWidget extends StatelessWidget {
     );
   }
 }
+
+class _HideKeyboardWidget extends StatelessWidget {
+  final void Function()? onTap;
+
+  final Function(bool) onHover;
+
+  final Widget child;
+
+  const _HideKeyboardWidget({
+    required this.onTap,
+    required this.onHover,
+    required this.child,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    if (PlatformInfos.isWeb) {
+      return MouseRegion(
+        child: child,
+        onHover: (event) {
+          onHover(true);
+        },
```

Comment:
how about to use it for mobile too?

Reply:
The widget is tracking the movement of `mouse`, so mobile don't have it

Reply:
Why I question it? Because right now, user can use keyboard and mouse with Android and iOS (Samsung Dex, mouse in iPad,....)